components/forms/banks/BankSearch.vue

```diff
@@ -1,11 +1,22 @@
 <template>
-  <div v-if="hasBanks && loaded && !disabled" v-clickaway="hideList">
-    <div class="relative">
+  <div class="col-span-2">
+    <BaseField
```

Comment:
using existing `BaseField` wrapper here so that we can reuse the warning and label functionality from there